app/src/main/java/com/hedvig/app/feature/tracking/LaunchTrackingInitializer.kt

```diff
@@ -0,0 +1,20 @@
+package com.hedvig.app.feature.tracking
+
+import android.content.Context
+import androidx.core.app.NotificationManagerCompat
+import androidx.startup.Initializer
+import com.hedvig.app.feature.di.KoinInitializer
+import com.hedvig.hanalytics.HAnalytics
+import org.koin.core.component.KoinComponent
+import org.koin.core.component.inject
+
+class LaunchTrackingInitializer : Initializer<Unit>, KoinComponent {
```

Comment:
I decided to go with the Jetpack App Startup for this type of 'track some stuff on launch'-events - it seemed like a good fit.